### DISCUSSION TITLE
Subheading

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#292] [Component suggestions]
<img width="480" alt="visuele weergave van de subheading component" src="https://github.com/user-attachments/assets/40494ca0-ebe6-4e70-a491-c4238602786f">

We willen graag meer leren over de **Subheading** component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, Github of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1728993900789659)

---

## Naam
Subheading

## Doel
Een ondersteunende tekst die direct na een kop staat.

## Gerelateerde componenten
[Pre-heading](https://github.com/orgs/nl-design-system/discussions/286), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289), [Heading Group](https://github.com/orgs/nl-design-system/discussions/323)

## Onderzoek
[What Is A Subheading and Why Are They Important? - Mailchimp](https://mailchimp.com/resources/what-is-a-subheading/)
Het artikel legt uit wat subheadings zijn en waarom ze belangrijk zijn voor zowel leesbaarheid als SEO.

[Visual Hierarchy in UX: Definition - Nielsen Norman Group](https://www.nngroup.com/articles/visual-hierarchy-ux-definition/)
Het artikel legt uit dat een duidelijke visuele hiërarchie het oog van de gebruiker naar de belangrijkste elementen op de pagina leidt. Deze hiërarchie kan worden gecreëerd door variaties in kleur en contrast, schaal en groepering. Zo wordt er specifiek genoemd dat er een verschil in schaal moet zijn tussen de Heading en Subheading om de inhoud overzichtelijk en goed leesbaar te maken.

**Comment by Rozerinay** [comment]
## DSO

- [Documentatie website]({link-naar-component-in-documentatie-website})
- [Storybook](https://storybook.dso-toolkit.nl/66.0.0/?path=/story/voorbeeldpagina-s-patronen-subtitel-subtitel-voor-een-definitie-lijst--subtitel-voor-een-definitie-lijst)
- [Github](https://github.com/dso-toolkit/dso-toolkit/tree/master/packages/dso-toolkit/src/components/subtitle)

---

### Screenshots / Varianten

Subtitel voor een Definitie Lijst
<img width="400" alt="image" src="https://github.com/user-attachments/assets/8b075ec2-8bb7-4566-8b12-363f8583665b">

Subtitel voor een Heading
<img width="400" alt="image" src="https://github.com/user-attachments/assets/f79f2272-27ac-4675-beb8-d4368bf7ddda">

Subtitel voor een Paragraaf
<img width="400" alt="image" src="https://github.com/user-attachments/assets/115964ac-8064-49ac-8caa-6700d54a2de0">

---

### Notities
- Onder de naam: Subtitel. 
- Subtitel is een `<p role="doc-subtitle">`, wat aangeeft dat de Paragraaf de rol van Subtitel vervult binnen het document.
- Font-weight: bold.

**Comment by Rozerinay** [comment]
## OpenGemeenten

- [Documentatie website](https://componenten.lenteveld.nl/follow-up-contactpersoon)
- [PatternLab](https://patternlab.lenteveld.nl/patterns/02-organisms-060-FollowUp-Components-20-FollowUp-Contact-OpenGemeentenPeoplePages-Followup/02-organisms-060-FollowUp-Components-20-FollowUp-Contact-OpenGemeentenPeoplePages-Followup.html)

---

### Screenshots / Varianten

**Documentatie website**

<img width="400" alt="image" src="https://github.com/user-attachments/assets/4f6e1cef-4cd6-48f7-818f-8d01d6e457ca">

**PatternLab**

<img width="400" alt="image" src="https://github.com/user-attachments/assets/e35d74c3-ca41-4999-868c-c26d9e55e19b">

---

### Notities
- Geen component gevonden.
- In de component ‘Follow-up contactpersoon’ wordt de naam van de contactpersoon weergegeven als een Heading-3, gevolgd door een Paragraph die fungeert als Subtitel.
- Font-weight: regular.

**Comment by AlineNap** [comment]
Ik vraag mij af of de subheading en pre heading een apart component zijn. Hergebruik je deze onderdelen en haar stijling op andere plekken dan de heading-group? Zo niet zou ik deze componenten onderdeel maken van de heading-group a la:

- prefix.heading-group.preheading.*
- prefix.heading-group.subheading.*

Net als een label in een button of accordion. 
Of de content en attribution in de blockqoute 

Zelfde [opmerking bij de pre-heading](https://github.com/orgs/nl-design-system/discussions/286#discussioncomment-10957377)

**Comment by Rozerinay** [comment]
## RVO

- [Documentatie website](https://nl-design-system.github.io/rvo/docs/components/hero/)
- [Storybook](https://nl-design-system.github.io/rvo/?path=/story/componenten-hero--default)
- [Figma](https://www.figma.com/design/Sj6myBL1Fvot5M1qGxzvEo/ROOS-(RVO-Design-System)?node-id=2734-2210&node-type=canvas&t=nb9U3toS0jKls1el-11)
- [Github](https://github.com/nl-design-system/rvo/tree/master/components/hero)

---

### Screenshots / Varianten

<img width="483" alt="image" src="https://github.com/user-attachments/assets/c4b4c666-4cb7-4a8d-be0b-61c24d90ad73">

---

### Notities
- Geen component gevonden.
- In de component 'Hero' zit een Subtitle. 
- Font-weight: regular.
- Font-style: italic.

**Comment by Rozerinay** [comment]
We sluiten de Discussion voor deze component, omdat deze niet langer een zelfstandige component is maar onderdeel uitmaakt van de [Heading Group](https://github.com/orgs/nl-design-system/discussions/323).
